src/root/Root.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import Root from './Root';
+
+describe('Root', () => {
+  it('renders without crashing', () => {
+    shallow(<Root />);
+  });
+
+  it('renders assignee list and header components', () => {
+    const wrapper = mount(<Root />);
+
+    expect(wrapper.find('.AssigneeList').length).toEqual(1)
+    expect(wrapper.find('.Header').length).toEqual(1)
+  });
```

Comment:
Thanks! It could look even better but I couldn't get this to work - from this [file](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#running-tests)

<img width="1007" alt="screen shot 2017-03-28 at 10 40 28 am" src="https://cloud.githubusercontent.com/assets/16868275/24416768/1ae47078-13a3-11e7-9bb1-5578ef6c5484.png">